job-service-container/pom.xml

```diff
@@ -137,11 +137,6 @@
             <artifactId>${caf.worker-queue.impl}</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
I'm not sure why the `job-service-container` integration tests need a file store. They don't use it as far as I can see, but when the tests startup they look for a data store implementation (they're not using the worker testing framework though, I'm not entirely sure how they work).

I thought I would go ahead and remove the worker-store-fs test dependency here anyway so that worker-store-http is used instead:

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running TestSuite
...
[main] INFO com.hpe.caf.util.ModuleLoader - Detected component implementation HttpDataStoreProvider
```